main/menu.js

```diff
@@ -150,12 +154,12 @@ function getMenuTemplate () {
             if (process.platform === 'darwin') return 'Ctrl+Command+F'
             else return 'F11'
           })(),
-          click: toggleFullScreen
+          click: () => toggleFullScreen()
```

Comment:
toggleFullScreen doesn't actually need a boolean argument, right?

I think only `onToggleFullScreen` is taking true or false. `toggleFullScreen()` is always called with no args afaik